.github/workflows/release-build-all.yml

```diff
@@ -0,0 +1,199 @@
+name: Release - Build All
```

Comment:
As this has the deployAll to main branch, can we please change the name to Build All / and Deploy to Dev. Also if possible please change the deployAll name to deployDev.

Reply:
I see the point, but the intention of the script is to build, the "deploy" part is just to keep to the current behavior of building and deploying for the `main` branch to ensure that it is "deployable". As we talked I would prefer to keep the name as it is for now.